src/styles/main.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@import "./utils/variables.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/header.scss";
+
+
+body,
+ul,
+p,
+div {
   margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
u can set `font-family` globally (only for `html` tag) and use `font-family: inherit` in child elements